features/fixtures/maze_runner/Assets/Scripts/Main.cs

```diff
@@ -220,26 +212,26 @@ void PrepareConfigForScenario(Configuration config, string scenario)
                 config.EnabledReleaseStages = new[] { "production" };
                 break;
             case "UncaughtExceptionAsUnhandled":
```

Comment:
I think it would make sense to update these scenarios to match the new terminology or it's going to cause a headache for some future dev (who may be you 😄 )